frontend/src/v5/ui/controls/formDatePicker/formDateTimePicker.component.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { DateTimePicker, DateTimePickerProps } from '@mui/x-date-pickers';
+import { formatMessage } from '@/v5/services/intl';
+import { FormBaseCalendarPicker, FormBaseCalendarPickerProps } from './formBaseCalendarPicker/formBaseCalendarPicker.component';
+
+type FormDateTimePickerProps = FormBaseCalendarPickerProps & Partial<DateTimePickerProps<any, any>>;
+export const FormDateTimePicker = (props: FormDateTimePickerProps) => {
+	const formatTime = (time) => time.replace('@', formatMessage({
+		id: 'form.dateTime.at',
+		defaultMessage: 'at',
+	}));
+
+	return (
+		<FormBaseCalendarPicker
+			PickerComponent={DateTimePicker}
+			inputFormat="DD/MM/YYYY @ hh:mma"
```

Comment:
Can you try to use the format in react-intl?
This should be handy
https://stackoverflow.com/questions/51749927/extract-the-date-format-from-react-intl

Reply:
I managed to fix the date, but not the time

Reply:
I managed to fix the date, but not the time